gobiko/apns/client.py

```diff
@@ -87,12 +87,12 @@ def send_bulk_message(self, registration_ids, alert, **kwargs):
         if not bad_registration_ids:
             return res
 
-        if not good_registration_ids:
+        elif not good_registration_ids:
             raise BadDeviceToken("None of the registration ids were accepted"
                                  "Rerun individual ids with ``send_message()``"
                                  "to get more details about why")
 
-        if bad_registration_ids and good_registration_ids:
+        else:
```

Comment:
The third `if` looks like `else` condition for two previous cases. I think it will be more readable to rewrite
```if A; if B; if not A and not B```
as
```if A; elif B; else```